game/src/info/trip.rs

```diff
@@ -501,28 +509,38 @@ fn make_bar(
     progress_along_path: Option<f64>,
 ) -> Widget {
     let map = &app.primary.map;
-    let data_inter_opt = &app
+    let intersection_delays = &app
+        .primary
+        .sim
+        .get_analytics()
+        .trip_intersection_delays
+        .get(&trip_id);
+    // TODO There surely has to be a better way of doing this?
```

Comment:
I haven't found it. `unwrap_or_else(BTreeMap::new)` would be ideal, but it's got to be an `&BTreeMap`, and if you try `unwrap_or_else(|| &BTreeMap::new())`, the scope of that is too small. But as Michael mentioned, you can declare the two blank maps first, then do `lane_speeds.unwrap_or(&blank_lane_speeds)`

Reply:
Yup changed